src/Faker/Provider/ko_KR/PhoneNumber.php

```diff
@@ -4,14 +4,51 @@
 
 class PhoneNumber extends \Faker\Provider\PhoneNumber
 {
+    //reference : https://ko.wikipedia.org/wiki/%EB%8C%80%ED%95%9C%EB%AF%BC%EA%B5%AD%EC%9D%98_%EC%A0%84%ED%99%94%EB%B2%88%ED%98%B8_%EC%B2%B4%EA%B3%84
+    
     protected static $formats = array(
-        '010-####-####',
+        //local area phone format
         '070-####-####',
         '02-####-####',
         '03#-####-####',
         '04#-####-####',
         '05#-####-####',
         '06#-####-####',
+        
+        //cell phone format
+        '010-####-####',
+        
+        //others: Intelligent Network(기간통신사업자)
         '1588-####',
+        '1577-####',
```

Comment:
due to the way `randomElement` works, these formats will be randomly picked way more often than the others. I suggest you use an intermediate formatter for those to avoid this distortion.